DESCRIPTION

```diff
@@ -56,5 +57,8 @@ Config/testthat/edition: 3
 Encoding: UTF-8
 Language: en-US
 LazyData: true
+remotes: 
+    tidymodels/yardstick#347,
+    r-lib/rlang
```

Comment:
This is to fix a bad link in a snapshot. Lionel thinks it will be on CRAN within 2 weeks.